proseco/core.py

```diff
@@ -398,6 +401,62 @@ def get_id_idx(self, id):
 
         return idx
 
+    def get_candidates_mask(self, stars):
+        """Return a boolean mask indicating which ``stars`` are acceptable candidates
+        for the parent class.  This basically applies to guide and acq, though
+        ACACatalog uses a hybrid of those two.
+
+        This is used directly in selection of candidates.  For the base class all
+        stars are considered acceptable.
+
+        :param stars: StarsTable of stars
+        :returns: bool mask
+
+        """
+        return np.ones(len(stars), dtype=bool)
+
+    @property
+    def bad_stars_mask(self):
+        """Mask of stars that are not acceptable candidates for the parent class.  This
+        basically applies to guide and acq, though ACACatalog uses a hybrid of
+        those two.
+
+        This is mostly used for plotting, corresponding to the ``bad_stars``
+        arg of plot_stars where bad stars are plotting in a distinctive color.
+
+        """
+        if not hasattr(self, '_bad_stars'):
+            self._bad_stars_mask = ~self.get_candidates_mask(self.stars)
+        return self._bad_stars_mask
+
+    @bad_stars_mask.setter
+    def bad_stars_mask(self, value):
+        self._bad_stars_mask = value
+
+    @property
+    def bad_stars(self):
+        """Back-compatibility property for bad_stars_mask"""
+        return self.bad_stars_mask
+
+    def plot(self, ax=None, **kwargs):
+        """
+        Plot the catalog and background stars.
+
+        :param ax: matplotlib axes object for plotting to (optional)
+        :param kwargs: other keyword args for plot_stars
+        """
+        from chandra_aca.plot import plot_stars
+        import matplotlib.pyplot as plt
+
+        kwargs.setdefault('stars', np.array([]))
+
+        # Explicitly set bad_stars so that plot_stars does not apply its
+        # internal version (which is no lot always the right answer).
+        kwargs.setdefault('bad_stars', np.zeros(len(kwargs['stars']), dtype=bool))
+
+        plot_stars(attitude=self.att, ax=ax, **kwargs)
```

Comment:
Looks good. Do we need more changes/options in chandra_aca.plot_stars?  Other colors etc?

Reply:
Maybe, but that's for another day.